### DISCUSSION TITLE
Windows builds failing

### DISCUSSION THREAD

**Discussion by oorzkws** [nicotine-plus/nicotine-plus#2087] [Q&A] [answered]
Hi, the builds under [actions](https://github.com/nicotine-plus/nicotine-plus/actions) have been failing for a few weeks. I don't have expertise with the packaging system to PR a fix myself, is there any possibility of this being resolved?

**Comment by slook** [comment]
Yes, the bug is known in a dependency, just it hasn't been decided what solution to take yet.

Reported here: https://github.com/marcelotduarte/cx_Freeze/issues/1510

**Reply by slook** [reply]
See https://github.com/nicotine-plus/nicotine-plus/pull/2089 :

> I'm shipping an unstable version of cx_Freeze until the fix makes it to the stable release.

**Comment by mathiascode** [comment, marked as answer]
This is resolved now.
